### ISSUE TITLE
[bitnami/nginx-ingress-controller] Is there a way to enable gzip or brotli text compression?

### ISSUE THREAD

**Issue by Huzaifa-zy** [bitnami/charts#6496]
I'm using the Bitnami NGINX Ingress Controller on GKE. I'm looking for a way to enable text compression using gzip or brotli. I've already gone through the official NGINX docs and added the required parameters to the ConfigMap on my K8s cluster. But that doesn't seem to do anything as my response headers don't contain a "content-encoded" header. I've double checked that the deployment.yaml is pointing to the correct ConfigMap and checked the response headers on multiple browsers.

I'm just curious if this feature is supported through the bitnami helm chart, even if it's just editing the parameter in the nginx.conf file inside the container.

**Comment by alemorcuq** [comment]
I'm checking this at the moment, I will get back to you soon.

**Comment by Huzaifa-zy** [comment]
I actually looked into it yesterday. I created a new ConfigMap and directed my deployment to use that ConfigMap specifically. Figured out that I was missing mentioning the namespace for my ConfigMap (it was default so I thought it should work without mentioning it). Regardless, I got it working for gzip but encountered another issue.

When I specified brotli compression in the ConfigMap, the Ingress controller pod started crashing. It would restart and stay on Running 0/1 then restart again. I reverted back to gzip and it started working again. Any idea why this could be?
